From 425f784fbb21e260605302ed5dee05f3e319f494 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Mon, 14 Aug 2017 12:07:28 -0500
Subject: [PATCH] start the service if battery exists

Signed-off-by: Simental Magana, Marcos <marcos.simental.magana@intel.com>
---
 data/thermald.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/thermald.service.in b/data/thermald.service.in
index 17d640a..0187f0d 100644
--- a/data/thermald.service.in
+++ b/data/thermald.service.in
@@ -1,5 +1,6 @@
 [Unit]
 Description=Thermal Daemon Service
+ConditionPathExistsGlob=/sys/class/power_supply/BAT*
 
 [Service]
 Type=dbus
-- 
2.14.1

